jplag/src/main/java/de/jplag/CommandLineArgument.java

```diff
@@ -112,4 +132,10 @@ private String retrieveDescriptionFromMessages() {
         }
         return Messages.getString(getClass().getSimpleName() + "." + builder.toString());
     }
+
+    /** Allowed number of values of an argument. */
+    public static enum NumberOfValues {
+        SINGLE_VALUE,
+        ONE_OR_MORE_VALUES,
+    }
```

Comment:
Even if it is short now, it might be extended in the future, so maybe make it non-nested.

Reply:
Done.

I was somewhat tempted to wrap a new sub-package `de.jplag.cli` around it (`CommandLineArgument`, `Cli`, and `NumberOfArgunmentValues` files) but not sure that was desired.

Could do it in the next patch though, I need to add the `--old` and `--new` options anyway.  
(`--old` and `--new ` is what we ended up with as replacement of the  `--checked` and `--unchecked` options, but I'll give you the full list and we can discuss it then.)

Reply:
Makes sense, but I don't need a new package that badly, probably better make a decision in the context of all code, and that's out of scope for me.